<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteFileWidget</class>
 <widget class="QWidget" name="RemoteFileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="1" colspan="2">
             <widget class="QComboBox" name="fileAtrribute">
              <item>
               <property name="text">
                <string>全部</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>解锁</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>锁定</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="QComboBox" name="videoType">
              <item>
               <property name="text">
                <string>全部</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>定时录像</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>移动侦测</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>报警触发</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>报警 | 移动侦测</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>报警 &amp; 移动侦测</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>命令触发</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>录像类型</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>文件属性</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QTimeEdit" name="startTime">
              <property name="displayFormat">
               <string>HH:mm:ss</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>开始时间</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>结束时间</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDateEdit" name="startDate"/>
            </item>
            <item row="3" column="2">
             <widget class="QTimeEdit" name="endTime">
              <property name="displayFormat">
               <string>HH:mm:ss</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDateEdit" name="endDate"/>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="findBtn">
              <property name="text">
               <string>开始查找</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>下载</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QPushButton" name="pushButton_9">
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>文件名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>文件大小</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="playFrame">
         <property name="styleSheet">
          <string notr="true">image:url(:/Homework/Resources/logo.png);
border:1px solid gray;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QSlider" name="processSlider">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QSlider
{
	min-height:5px;
	max-height:5px;
    border:5px solid rgba(255, 255, 255, 0);
    }
QSlider::add-page:horizontal
{     
   background-color:rgb(87, 97, 106);
   height:4px;
}
QSlider::sub-page:horizontal 
{
   background-color: rgb(225, 75, 0);
	height:4px;
 }
    
QSlider::groove:horizontal 
{
	background:transparent;
	height:6px;
}    
QSlider::handle:horizontal     
{     
	height: 13px;
	width:13px;
    border-image: url(:/Homework/Resources/ic_volume_thumb.png);
    margin: -4 -0px; 
}</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="stopBtn">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/*QPushButton 
{
	border-image: url(:/Homework/Resources/play1.png);
}

QPushButton:pressed
{
	border-image: url(:/Homework/Resources/play2.png);
}

QPushButton:hover
{
	border-image: url(:/Homework/Resources/play3.png);
}
*/</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Homework.qrc">
             <normaloff>:/Homework/Resources/stop.png</normaloff>:/Homework/Resources/stop.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="slowPlayBtn">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Homework.qrc">
             <normaloff>:/Homework/Resources/before.png</normaloff>:/Homework/Resources/before.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="playBtn">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/*QPushButton 
{
	border-image: url(:/Homework/Resources/play1.png);
}

QPushButton:pressed
{
	border-image: url(:/Homework/Resources/play2.png);
}

QPushButton:hover
{
	border-image: url(:/Homework/Resources/play3.png);
}
*/</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Homework.qrc">
             <normaloff>:/Homework/Resources/play.png</normaloff>:/Homework/Resources/play.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fastPlayBtn">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Homework.qrc">
             <normaloff>:/Homework/Resources/next.png</normaloff>:/Homework/Resources/next.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/Homework/Resources/voice.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="voiceSlider">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QSlider
{
	min-height:5px;
	max-height:5px;
    border:5px solid rgba(255, 255, 255, 0);
    }
QSlider::add-page:horizontal
{     
   background-color:rgb(87, 97, 106);
   height:4px;
}
QSlider::sub-page:horizontal 
{
   background-color:  rgb(203, 203,203);
	height:4px;
 }
    
QSlider::groove:horizontal 
{
	background:transparent;
	height:6px;
}    
QSlider::handle:horizontal     
{     
	height: 13px;
	width:13px;
    border-image: url(:/Homework/Resources/ic_volume_thumb.png);
    margin: -4 -0px; 
}</string>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="sliderPosition">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Homework.qrc"/>
 </resources>
 <connections/>
</ui>
